conference/domain/pom.xml

```diff
@@ -108,5 +108,12 @@
         </dependency>
         <!-- logging end -->
 
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <version>2.6.1</version>
+        </dependency>
```

Comment:
Do not keep version here, but put this entry also in dependency management section of parent pom.xml and keep version there.
Have you also remove dbcp dependency?